src/Apps/Order/Components/ShippingAndPaymentReview.tsx

```diff
@@ -30,7 +34,7 @@ export const ShippingAndPaymentReview = ({
       </StepSummaryItem>
     ) : (
       <StepSummaryItem onChange={onChangeShipping} title="Shipping address">
-        <ShippingAddress {...address} />
+        <ShippingAddress {...address as ShippingAddressProps} />
```

Comment:
hmm I'd prefer not to see this cast, but I'll leave it for now because I'm planning to refactor the types here some more after this is merged.

Reply:
Yeah, the issue was that `ShippingAddressProps` are all non-null, but come as nullable from the schema. I think changing the `ShippingAddress` component would make sense.